api/nakadi-event-bus-api.yaml

```diff
@@ -1269,15 +1269,20 @@ definitions:
         description: configuration settings for kafka storage. Only necessary if the storage type is 'kafka'
         type: object
         properties:
+          exhibitor_address:
+            description: the Zookeeper address
+            type: string
+          exhibitor_port:
+            description: the Zookeeper path
+            type: string
           zk_address:
             description: the Zookeeper address
             type: string
           zk_path:
             description: the Zookeeper path
             type: string
         required:
-          - zk_address
-          - zk_path
+          - exhibitor_address
```

Comment:
I don't get it. I thought that it is important to have either (zk_address, zk_path) pair or (exhibitor_address, exhibitor_port, zk_path). In fact - no required fields at all. But having exhibitor_address required is actually incorrect. 
Am I correct?